src/plugins/workspace/public/components/workspace_initial/workspace_initial.tsx

```diff
@@ -69,7 +73,7 @@ export const WorkspaceInitial = () => {
             defaultMessage: 'Create a workspace',
           })}
           description={
-            <EuiToolTip content={isDashboardAdmin || noAdminToolTip}>
+            <EuiToolTip content={isDashboardAdmin ? <></> : noAdminToolTip}>
```

Comment:
```suggestion
            <EuiToolTip content={isDashboardAdmin ? null : noAdminToolTip}>
```
What about we just render null?

Reply:
updated